include/clang/AST/Expr.h

```diff
@@ -4814,13 +4814,24 @@ class RangeBoundsExpr : public BoundsExpr {
 private:
   enum { LOWER, UPPER, END_EXPR };
   Stmt *SubExprs[END_EXPR];
+  RelativeBoundsClause *DefaultClause;
   RelativeBoundsClause *RelativeClause;
+  /// Relative alignment base : bounds(lb, ub) rel_align(T)
+  /// compiler checkes whether sizeof(referent-type(base)) is common factor of
```

Comment:
Sorry for mistake
This update related to BoundsClause is wrongly merged into current branch
By merging branches, I had some mistakes
I will exlucde this updates for this commit